platform/opensuse/kanidm-unixd-tasks.service

```diff
@@ -3,8 +3,10 @@
 
 [Unit]
 Description=Kanidm Local Tasks
-After=chronyd.service ntpd.service network-online.target suspend.target kanidm-unixd.service
-Wants=kanidm-unixd.service
+After=chronyd.service ntpd.service network-online.target kanidm-unixd.service
+# This prevents starting unixd-tasks before unixd is running and
+# has created the socket necessary for communication.
+ConditionPathExists=/var/run/kanidm-unixd/task_sock
```

Comment:
tasks polls for this to exist, so this isn't strictly needed.

Reply:
Turns out it actually is. The unit is invalid if the path referred to by ReadWritePaths doesn't exist. That was the root cause of the strange namespace errors that delayed launch.

Reply:
Oh neat. Okay, then I stand corrected :)

Reply:
You may need to actually comment that fact then btw. That it influence rw paths.